daemons/Makefile

```diff
@@ -6,6 +6,7 @@ dss-sync dss-index:
 	git clean -df $@/domovoilib
 	cp -R ../dss ../dss-api.yml $@/domovoilib
 	./build_deploy_config.sh $@ $(STAGE)
+	if [[ $@ == "dss-index" ]]; then ./deploy_aws_elasticsearch.sh $@ $(STAGE); fi
```

Comment:
if this can be made multiline, please do so.  i forget if this is a restriction of makefiles though...